content/docs/concepts/architecture/index.md

```diff
@@ -64,8 +68,14 @@ This category of services reacts on Keptn events and perform: (1) initial tasks
 
```

Comment:
Can you write a sentence that the `gatekeeper-service` is also responsible for managing approvals?

Reply:
I have updated the description for the gatekeeper service. However, it is not responsible for managing the open approvals anymore, since that is now handled by the shipyard controller, using the generic task management